app/lib/project/ImportController.js

```diff
@@ -75,8 +76,11 @@ define([
                     return true;
                 }.bind( null, this._master )
             };
-            this._sourceGlyphSet = this._project.getGlyphSet(
```

Comment:
There no use of it left. But maybe it shoud not be deleted yet, but instead refactored:

On one hand, getGlyphSet deals with detecting UFO version.
On the other hand it indirectly operates on a hardcoded I/O object, instead of explicitely receiving an arbitrary I/O object as a parameter.

We seem to need to refactore it to both take care of UFO version detection as well as receiving an io module as parameter.